mesonbuild/coredata.py

```diff
@@ -108,6 +108,7 @@ def toint(self, valuestring):
 class UserUmaskOption(UserIntegerOption):
     def __init__(self, name, description, value, yielding=None):
         super().__init__(name, description, 0, 0o777, value, yielding)
+        self.choices = ['preserve', '0000-0777']
```

Comment:
This, too, is a new feature which the PR description says should not be here.

Reply:
How is that a new feature?

Reply:
NVM, brain fart.